### DISCUSSION TITLE
Device supported but provided by another manufacturer not supported

### DISCUSSION THREAD

**Discussion by PacmanForever** [Koenkk/zigbee2mqtt#12395] [Devices]
Hi eveyone,

this device is supported:

https://www.zigbee2mqtt.io/devices/ZVG1.html

I have the same but provided by another manufacturer (SASWELL). 

Zigbee2mqtt show this:

"Received message from unsupported device with Zigbee model 'TS0601' and manufacturer name '_TZE200_81isopgh'"

Is there any way to reuse current supported device? 
Can anyone (or me) copy device adapter and replace the manufacturer/model?

Thanks

**Comment by Koenkk** [comment]
Added.

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by PacmanForever** [reply]
Thank you very much @Koenkk

**Comment by MissK2000** [comment]
Hi @jvcsw ,
Would you be kind enough to tell me if this dev version shows the water consumption too ? I did manage to adapt the RTX file so that I can control my saswell valve but the water consumption is stuck to zero :(
Thanks a lot

**Reply by fliespl** [reply]
@MissK2000 did you ever solve this? Having same issue.

**Comment by PacmanForever** [comment]
My valve does not report the water consumed.
I created this issue weeks ago with no response.

https://github.com/Koenkk/zigbee2mqtt/issues/17011

**Reply by fliespl** [reply]
@jvcsw are you sure? I thought the same and then realized that it indeed reports on my end (AT THE END of watering with most current z2m version) but goes back to zero after some time (which is natural since it reports last usage). You can use helper to increase on each change.

**Reply by PacmanForever** [reply]
Yes, open and closed now:

![imagen](https://github.com/Koenkk/zigbee2mqtt/assets/37216513/d0c2e9fa-a8c2-4644-8445-04448610dd03)
